### ISSUE TITLE
spring boot 1.x mapping result handle(in UI) maybe wrong?

### ISSUE THREAD

**Issue by lagudomeze** [codecentric/spring-boot-admin#2122]
frontend mappings index.vue: isOldMetrics is true only if    **contentType equals "vnd.spring-boot.actuator.v2"**
https://github.com/codecentric/spring-boot-admin/blob/fd65e38e8b5fb54bab49c698358ec8df31866c62/spring-boot-admin-server-ui/src/main/frontend/views/instances/mappings/index.vue#L93-L97

but with legacy convert to **last spring boot api version** 

https://github.com/codecentric/spring-boot-admin/blob/fd65e38e8b5fb54bab49c698358ec8df31866c62/spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/web/client/InstanceExchangeFilterFunctions.java#L142-L147

![image](https://user-images.githubusercontent.com/18391140/190886671-1386b5b9-8b6c-42d1-a534-d8c77869bd54.png)

As a result mapping is always old metric，convert is useless！

**Comment by lagudomeze** [comment]
maybe just change [mappings/index.vue#L93](https://github.com/codecentric/spring-boot-admin/blob/master/spring-boot-admin-server-ui/src/main/frontend/views/instances/mappings/index.vue#L93)  to v3 is fine?

**Comment by SteKoe** [comment]
Hi @lagudomeze,

thanks for filing this issue and providing a PR 😍! 
We have checked your inquiry and tested it using Spring Boot 1.4 up to 3.0. Sadly, your change breaks accessing metrics on Spring Boot services using versions 2.0 and 2.1. Nevertheless, we have implemented your suggested change in a slightly different way and are now checking both `ApiVersions` (V2 -> Spring Boot 2.0/2.1, V3 -> Spring Boot >2.2). 

Regards,
SteKoe
